docs/src/development/troubleshoot.md

```diff
@@ -212,46 +212,14 @@ Make sure that the paths for files like media files, dependencies, and databases
 If large files are already in the repository, the open-source tool [bfg-repo-cleaner](https://rtyley.github.io/bfg-repo-cleaner/)
 can assist in cleaning up the repository by purging older commits, removing unnecessary files, and more.
 
-If none of these suggestions work, create a [support ticket](https://console.platform.sh/-/users/~/tickets/open).
-
-## Stuck build or deployment
-
-If you see a build or deployment running longer than expected, it may be one of the following cases:
-
-* The build is blocked by a process in your `build` hook.
-* The deployment is blocked by a long running process in your `deploy` hook.
-* The deployment is blocked by a long running cron job in the environment.
-* The deployment is blocked by a long running cron job in the parent environment.
-
-To determine if your environment is being stuck in the build or the deployment, check your [activity log](../increase-observability/logs/access-logs.md#activity-logs).
-
-If the activity has the result `success`, the build has completed successfully and the system is trying to deploy.
-If the result is still `running`, the build is stuck.
-
-In most regions, stuck builds terminate after one hour.
-To have the build killed in [legacy regions](./regions.md#legacy-regions), create a [support ticket](https://console.platform.sh/-/users/~/tickets/open).
-
-When a _deployment_ is blocked, you should try the following:
-
-1. Connect to your environment using [SSH](./ssh/_index.md).
-2. Find any long-running cron jobs or deploy hooks on the environment by running `ps afx`.
-3. Kill any long running processes with `kill <PID>`.
-  Replace `<PID>` with the process ID shown by `ps afx`.
-
-If a `sync` of `activate` process is stuck, try the above on the parent environment.
-
-## Slow or failing build or deployment
-
-Builds can take long time or fail.
-Most of the time, it's related to an application issue.
-Here are a few tips that can help you find the exact cause.
+If none of these suggestions work, create a [support ticket](../overview/get-support.md).
```

Comment:
The comment was intentionally on many lines, not this one. It wasn't about the link to the support ticket but the large moving of sections. Putting "stuck build" inside "build hooks" makes it about hooks, which isn't necessarily how users would see it. It might be a good idea, but this PR doesn't seem like the place to be making such changes.